apps/web/src/pages/activities/hooks/useNotificationStatus.ts

```diff
@@ -5,8 +5,9 @@ export const useNotificationStatus = (notification: any): JobStatusEnum => {
   const [status, setStatus] = useState(JobStatusEnum.PENDING);
 
   useEffect(() => {
-    const result = notification?.jobs
```

Comment:
I think this should happen in the backend, so the API actually returns an array not undefined here